packages/editor/src/store/defaults.js

```diff
@@ -10,6 +10,7 @@ import { SETTINGS_DEFAULTS } from '@wordpress/block-editor';
  * @property {boolean}       richEditingEnabled    Whether rich editing is enabled or not
  * @property {boolean}       codeEditingEnabled    Whether code editing is enabled or not
  * @property {boolean}       enableCustomFields    Whether the WordPress custom fields are enabled or not.
+ * @property {boolean}       fontLibraryEnabled    Whether the font library is enabled or not.
```

Comment:
It makes my brain hurt that I've seen four different ways of naming similar boolean properties: `enableSettingName`, `disableSettingName`, `settingNameEnabled`, and just `settingName`. 

But I guess there's nothing we can do about that at this point, so I'm okay with the way you've constructed it.

Reply:
:D We are following the major number of examples from that file.